app/src/main/res/layout/fragment_home.xml

```diff
@@ -16,6 +16,7 @@
         android:textAppearance="@style/TextAppearance.AppCompat.Display1"
         android:textColor="@color/colorPrimaryDark"
         android:textSize="32sp"
+        android:hint="Add a new report."
```

Comment:
Hint is a good idea, and it saves the UI from one more widget: a label.  I'd expand on what you have here.  Instead of hardcoded text, put this in strings.xml, so that it can be internationalized.